### PR TITLE
rm: add German translation

### PR DESCRIPTION
**Pull request by skyjx**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).

### CODE REVIEW

**Review by blueskyson — APPROVED** [review]

File: pages.de/common/rm.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+`rm -r {{pfad/zu/verzeichnis}}`
+
+- Erzwinge das Löschen eines Verzeichnis, ohne Eingabeaufforderung zur Bestätigung oder Anzeigen von Fehlermeldungen:
+
+`rm -rf {{pfad/zu/verzeichnis}}`
+
+- Lösche mehrere Dateien mit Eingabeaufforderung zur Bestätigung für jede Datei:
+
+`rm -i {{datei(en)}}`
```
Comment (EmilySeville7cfg) [inline pages.de/common/rm.md:20]:
```suggestion
`rm -i {{datei(en)}}`
```

Mandatory fix (error source: [original page](https://github.com/tldr-pages/tldr/blob/main/pages/common/rm.md)): [wrong path placeholder format](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md#paths).
Note: #9578 addresses all such issues automatically.

**Review by EmilySeville7cfg — CHANGES_REQUESTED** [review]
- Mandatory to fix errors found, changes requested.
- Thanks for a contribution! ❤️

File: pages.de/common/rm.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+- Lösche Dateien an beliebigen Speicherorten:
+
+`rm {{pfad/zu/datei}} {{pfad/zu/anderer/datei}}`
+
+- Lösche ein Verzeichnis und alle seine Unterverzeichnisse rekursiv:
+
+`rm -r {{pfad/zu/verzeichnis}}`
+
+- Erzwinge das Löschen eines Verzeichnis, ohne Eingabeaufforderung zur Bestätigung oder Anzeigen von Fehlermeldungen:
```
Comment (pixelcmtd) [inline pages.de/common/rm.md:14]: https://www.duden.de/deklination/substantive/Verzeichnis

```suggestion
- Erzwinge das Löschen eines Verzeichnisses, ohne Eingabeaufforderung zur Bestätigung oder Anzeigen von Fehlermeldungen:
```

File: pages.de/common/rm.md (hunk 1 above)
Comment (pixelcmtd) [inline pages.de/common/rm.md:20]: Why did you mark @EmilySeville7cfg's comment as resolved without explaining or changing anything? We are changing the other versions of this page, what is your rationale for not adopting it here?

```suggestion
`rm -i {{pfad/zu/datei1 pfad/zu/datei2 ...}}`
```

Comment (skyjx) [reply pages.de/common/rm.md:20]: When I first saw the comment I could not figure out what was meant, and I did not find a difference in other versions either. I am sorry for not asking.

**Review by EmilySeville7cfg — APPROVED** [review]

File: pages.de/common/rm.md (hunk 1 above)
Comment (EmilySeville7cfg) [reply pages.de/common/rm.md]: > When I first saw the comment I could not figure out what was meant, and I did not find a difference in other versions either. I am sorry for not asking.

I can mean that I have to describe issues better.

**Review by pixelcmtd — APPROVED** [review]

**Pull request merged**